kafka-rest/src/main/java/io/confluent/kafkarest/common/KafkaFutures.java

```diff
@@ -40,6 +44,7 @@ public static <T> CompletableFuture<T> toCompletableFuture(KafkaFuture<T> kafkaF
           if (exception == null) {
             completableFuture.complete(value);
           } else {
+            log.error("Caught Exception when Complete KafkaFuture", exception);
```

Comment:
Caught Exception when Complete KafkaFuture
->

Caught Exception on completion of KafkaFuture